src/components/Modal.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {FormEvent, useState} from 'react'
+import './Modal.scss'
```

Comment:
to achieve local scoping, the file needs to be a `Modal.module.css`. otherwise there might be some weird overrides in other components in near future